pkg/proxy/client/client_conn.go

```diff
@@ -77,7 +81,10 @@ func (cc *ClientConnection) Run(ctx context.Context) {
 	}
 
 	if err := cc.processMsg(ctx); err != nil {
-		cc.logger.Info("process message fails", zap.String("remoteAddr", cc.Addr()), zap.Error(err))
+		clientErr := errors.Is(err, ErrClientConn)
+		if !(clientErr && pnet.IsDisconnectError(err)) {
```

Comment:
If `clientErr && pnet.IsDisconnectError(err) == true`, it's still not printed.

Reply:
So why not ignore `client disconnect` errors?  Even if you print it out, you still need to search all files to conclude that there is no `server disconnect` error by realizing all errors are from `client`.

I'll give another modification, `EOF` is a normal disconnection state that should be ignored. And `broken pipe` or `connection reset` will be logged.

Reply:
BTW, there is another argument. Since server will nerver end the connection in the command phase, thus disconnection must come from clients or network errors. Now all network errors are logged. If no errors are logged, then it is  disconnected by`EOF` from clients obviously.

Reply:
I don't think `EOF` is normal. Sending a `STMT_QUIT` is normal.

Reply:
It came up to me that there's always another disconnection log. So ignoring the `EOF` may be fine.